packages/next-ui/Layout/components/LayoutHeaderContent.tsx

```diff
@@ -192,7 +192,12 @@ export function LayoutHeaderContent(props: LayoutHeaderContentProps) {
             },
           })}
         >
-          <MotionDiv layout={layout}>{children}</MotionDiv>
+          <MotionDiv
+            sx={() => ({ minWidth: 0 /* required for ellipsis to work */ })}
```

Comment:
```suggestion
            sx={{ minWidth: 0 }}
```

No callback needed and the comment can be derived from the commit message.